src/test/java/org/apache/commons/collections4/bloomfilter/hasher/HasherBuilderTest.java

```diff
@@ -58,10 +62,9 @@ public Builder with(byte[] item) {
     public void withCharSequenceTest() {
         final String ascii = "plain";
         final String extended = getExtendedString();
-        for (final String s : new String[] {ascii, extended}) {
-            for (final Charset cs : new Charset[] {
-                StandardCharsets.ISO_8859_1, StandardCharsets.UTF_8, StandardCharsets.UTF_16
-            }) {
+        for (final String s : new String[] { ascii, extended }) {
+            for (final Charset cs : new Charset[] { StandardCharsets.ISO_8859_1, StandardCharsets.UTF_8,
+                StandardCharsets.UTF_16 }) {
                 TestBuilder builder = new TestBuilder();
```

Comment:
If you are using Eclipse to format the entire file then there is a problem with your Eclipse formatting since it changed what was already in the file. There is no formatting file that has been applied to the collections source code. All formatting is done manually and must pass the checkstyle rules. These rules are not as strict as they could be but that is for another discussion.

In general you should only format new additions to existing source. If you reformat the existing source then you are adding noise to the PR.